README.md

```diff
@@ -288,6 +288,90 @@ public class PulsarErrorHandler {
 }
 ```
 
+#### 5. Reactor support (Flux)
```

Comment:
Great explanations. I wonder if you also have an example project. I think it would be nice if that would be one that uses the new API with this PR / different repo

Reply:
Hi @almogtavor, sure there is an example project. However, library needs to be released on Maven repo first in order to be used in example project. 

Link: https://github.com/majusko/java-pulsar-example

Reply:
That's why I've suggested doing this in the same repository. Anyway, it's great to see this project. Looking forward to the reactive streams-based example after the release.

Reply:
I see. I will create an example right after the release and let you know. Thank you for the feedback.